queries.sql

```diff
@@ -9,3 +9,30 @@
  select * from animals where name !='Gabumon';
  select * from animals where weight_kg >=10.4 and weight_kg <=17.3;
 
+begin;
+update animals set species= 'unspecified';
+rollback;
+
+ begin;
+ update animals set species= 'digimon' where name like '%mon';
+ update animals set species='pokemon' where species is null;
+ commit;
+
+ begin;
+ delete from animals;
+ rollback;
+
+  begin;
+  delete from animals where date_of_birth > '01-01-2022';
+  savepoint first_savepoint;
+  update animals set weight_kg = weight_kg * (-1);
+  rollback to first_savepoint;
+  update animals set weight_kg = weight_kg * (-1) where weight_kg <0;
+  commit;
+
+  select count(name) from animals;
+  select  count(name) from animals where escape_attempts = 0;
+  select AVG(weight_kg) from animals;
+  select MAX(escape_attempts) from animals where neutered = true or neutered = false;
+  select max(weight_kg), min(weight_kg) from animals group by species;
+  select AVG(escape_attempts) from animals where date_of_birth between '01-01-1990' and '01-01-2000' group by species;
```

Comment:
- [ ] Great work writing all the queries so far, for this last query, the requirement states those born between 1990 and 2000. This means the query should begin from ```'01-01-1990'``` and end at ```'31-12-2000'```